tagstudio/src/qt/ts_qt.py

```diff
@@ -814,7 +810,7 @@ def run_macro(self, name: MacroID, grid_idx: int):
                 )
 
         elif name == MacroID.BUILD_URL:
-            url = TagStudioCore.build_url(entry.id, source)
+            url = TagStudioCore.build_url(entry, source)
```

Comment:
Note: This change is also proposed in #612

Reply:
now that #612 has been merged, this could be removed

Reply:
I’ve rebased to the latest main and removed those changes.